samples/graph-chat-lifecycle/nodejs/src/manifest.json

```diff
@@ -0,0 +1,52 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.9/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.9",
+    "version": "1.0.1",
+    "id": "<<YOUR-MICROSOFT-APP-ID>>",
```

Comment:
Could you please move manifest file and images to separate folder?

Reply:
Moved manifest.json and images to manifest folder in src.